public/api-merged.json

```diff
@@ -15113,10 +15096,10 @@
       "type": "object",
       "properties": {
         "dashboard": {
-          "$ref": "#/definitions/Json"
+          "$ref": "#/definitions/dashboard"
         },
         "meta": {
-          "$ref": "#/definitions/Json"
+          "$ref": "#/definitions/dashboard"
```

Comment:
it should not be a definitions/dashboard here I think, read the logic, did you forget to regenerate with the new merge_specs.go maybe?

Reply:
👍 good point!